core/includes/install.inc

```diff
@@ -808,7 +853,8 @@ function backdrop_install_config_location() {
 
   if ($settings['config_active_class'] === 'ConfigFileStorage') {
     $is_empty = empty($config_directories['active']);
-    $is_default_file_storage = $config_directories['active'] === 'files/config_' . md5($database) . '/active';
+    $database_string = is_array($database) ? implode(',', $database) : $database;
```

Comment:
Let's use `serialize()` here as a sure-fire way to convert an array into a string, as there could conceivably be nested array values at some point.

```suggestion
    $database_string = is_array($database) ? serialize($database) : $database;
```

Reply:
Thanks @quicksketch .  I have used this change to simplify as we can simply `serialize` the `$database` variable regardless of whether it is a string or an array, so I can drop the `$database_string` step.